tests/common/common.go

```diff
@@ -28,3 +34,16 @@ func GenerateUID() string {
 
 	return string(b)
 }
+
+func SetCfgName(cfg *v1alpha1.Environment) {
+	sha := os.Getenv("GITHUB_SHA")
+	attempt := os.Getenv("GITHUB_RUN_ATTEMPT")
+	// short sha
+	if len(sha) > 8 {
+		sha = sha[:8]
+	}
+	// uid is unique for each run
+	uid := GenerateUID()
+
+	cfg.Name = fmt.Sprintf("ci%s-%s-%s", attempt, sha, uid)
```

Comment:
I am not sure if this change is desirable. Won't the names now become a lot longer?

Reply:
@tariq1890 

old code:-  ` opts.cfg.Name = opts.cfg.Name + "-" + common.GenerateUID()`

changed to :- 
`common.SetCfgName(&opts.cfg)

`

No, it will not be longer, it will align with our e2e tests for the lib/app (e.g., gpu-operator, gpu-driver-container, etc.), which follow the naming format ci-commitid-uuid.

Currently, holodeck e2e tests use names derived from the yaml file's given name plus a UUID (e.g., holodeck-uuid). 

As a result, the cleanup script, which only works for instances named in the format ci-commitid-uuid, will not clean up these instances. 

This change will ensure that any instances not properly cleaned up will now be handled correctly.